src/actors/rqgActorSheet.ts

```diff
@@ -1062,6 +1062,26 @@ export class RqgActorSheet extends ActorSheet<
           }
         });
       });
+
+    // Add Passion button
+    htmlElement?.querySelectorAll<HTMLElement>("[data-passion-add]").forEach((el) => {
+      el.addEventListener("click", async () => {
+        console.log("CLICK");
+        const defaultItemIconSettings: any = getGame().settings.get(
+          systemId,
+          "defaultItemIconSettings"
+        );
+        const newPassionName = localize("RQG.Actor.Passion.NewPassionName");
+        let passion = {
+          name: newPassionName,
+          type: "passion",
+          img: defaultItemIconSettings["passion"],
+          system: { passion: newPassionName },
+        };
+        //@ts-ignore
+        await Item.createDocuments([passion], { parent: this.actor });
```

Comment:
I think it would be good to open the itemsheet when the item is created. This can be done by adding `renderSheet: true` to the options.

Reply:
Done